tap_github/__init__.py

```diff
@@ -1143,20 +1143,23 @@ def get_all_projects_v2(schemas, repo_path, state, mdata, _start_date):
             'acctObjectName': acctObjectName,
         }
 
-        for projects in authed_graphql_all_pages(stream_name, query_template, query_values, path):
-            # store in cache for later usage in child streams
-            projects_v2.extend(projects)
+        try:
```

Comment:
This is just wrapped in a try catch, so no actual changes to the logic inside this block